pkg/kube/snapshot/snapshot_alpha.go

```diff
@@ -318,24 +317,23 @@ func (sna *SnapshotAlpha) CreateContentFromSource(ctx context.Context, source *S
 	return nil
 }
 
+func isReadyToUseAlpha(us *unstructured.Unstructured) (bool, error) {
+	vs := v1alpha1.VolumeSnapshot{}
+	if err := TransformUnstructured(us, &vs); err != nil {
+		return false, err
+	}
+	// Error can be set while waiting for creation
+	if vs.Status.Error != nil {
+		return false, errors.New(vs.Status.Error.Message)
+	}
+	return (vs.Status.ReadyToUse && vs.Status.CreationTime != nil), nil
+}
+
 // WaitOnReadyToUse will block until the Volumesnapshot in namespace 'namespace' with name 'snapshotName'
 // has status 'ReadyToUse' or 'ctx.Done()' is signalled.
+// FIXME: merge this with beta wait
```

Comment:
I think we should eventually add a `stable` wait and merge both alpha and beta into it

Reply:
I did actually merge the wait functions so this one is addressed.